### ISSUE TITLE
can not running on local URL

### ISSUE THREAD

**Issue by Pbugyon** [lllyasviel/IC-Light#17]
![Screenshot (19)](https://github.com/lllyasviel/IC-Light/assets/157469511/4767c945-999c-4fab-8091-e3ec0458ef3e)
Hi, when I try to access the local site I can't load the page, my browser (it doesn't matter which one because all the browsers I've tried give the same message) reports this error: "
Unable to reach site The web page at http://0.0.0.0:7860/ may be temporarily unavailable or has been permanently moved to a new web address.
ERR_ADDRESS_INVALID "

**Comment by muxin159** [comment]
I encountered the same problem, and finally, by modifying the last line of code in the "<img width="730" alt="97abe43f99d6943fa41d0b5fdabaccc" src="https://github.com/lllyasviel/IC-Light/assets/122255849/ec460abd-fb04-4ee3-a478-fd506a3e2042">" file from 0.0.0.0 to 127.0.0.1, I successfully accessed the page. The origin of 127.0.0.1 is the IP address of my computer—I hope this is helpful to you.

**Comment by muxin159** [comment]
<img width="730" alt="97abe43f99d6943fa41d0b5fdabaccc" src="https://github.com/lllyasviel/IC-Light/assets/122255849/4165fbca-48b4-4899-8faf-3beca0882dc2">

I encountered the same problem, and finally, by modifying the last line of code in the "gradio_demo.py" file from 0.0.0.0 to 127.0.0.1, I successfully accessed the page. The origin of 127.0.0.1 is the IP address of my computer—I hope this is helpful to you.

**Comment by baban** [comment]
@muxin159 
Tanks, I can fix,too.

**Comment by pynooblin** [comment]
url this: 
http://127.0.0.1:7860/

**Comment by Pbugyon** [comment]
> <img alt="97abe43f99d6943fa41d0b5fdabaccc" width="730" src="https://private-user-images.githubusercontent.com/122255849/329216311-4165fbca-48b4-4899-8faf-3beca0882dc2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNDgwMjcsIm5iZiI6MTcxNTM0NzcyNywicGF0aCI6Ii8xMjIyNTU4NDkvMzI5MjE2MzExLTQxNjVmYmNhLTQ4YjQtNDg5OS04ZmFmLTNiZWNhMDg4MmRjMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMFQxMzI4NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZDJmNzE4YmY5OTg2ZWFkYWZmMWMwMjUzNzU1MDI1MGUxYmRhMGVkNjg1NmRiZWQxNGM1OTA4ZmNiN2JiZWUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AYfEpzNRwsRGq6FPxz6qgcumkgJ2LnoF1osjaj13uJ0">
> I encountered the same problem, and finally, by modifying the last line of code in the "gradio_demo.py" file from 0.0.0.0 to 127.0.0.1, I successfully accessed the page. The origin of 127.0.0.1 is the IP address of my computer—I hope this is helpful to you.

yes I did it too but instead of what you wrote I entered "localhost"

**Comment by foxbeing7** [comment]
config server_name and server_port in block.launch
